### ISSUE TITLE
Suggestion: option the add grade

### ISSUE THREAD

**Issue by mormoshe** [asaf-kali/coursist#66]
Hey,
First, great project! 
I think that it will be useful if you'll add an option to add grades and calculate average for each course.

**Comment by evyatar100** [comment]
I don't think that a lot of people would do that - and thus the average would be off... But we defenetly need a way to show statistics.

**Comment by asaf-kali** [comment]
But we are planning on adding this feature for self use (for keeping track of the degree progress).

**Comment by evyatar100** [comment]
na
